src/lib/secureStorage.ts

```
@@ -0,0 +1,259 @@
+}
+
+// Export with specific key for borrower draft
+export const secureStorage = {
+  setDraft: async (data: any): Promise<void> => {
+    try {
+      await SecureStorage.set(STORAGE_KEY, data)
+    } catch (error) {
+      console.error('Failed to save draft securely:', error)
+      // Fallback to plain storage if encryption fails
+      if (typeof window !== 'undefined' && window.localStorage) {
+        localStorage.setItem(STORAGE_KEY + '_fallback', JSON.stringify(data))
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)